### DISCUSSION TITLE
release planning

### DISCUSSION THREAD

**Discussion by mtdvlpr** [sircharlo/meeting-media-manager#389] [General]
Hi @sircharlo, I was wondering when you are planning on releasing a new version. It's been a while and you promised me a version 22.5.8 where the Linux version would be fixed. To be clear, I don't mind waiting. I'm on vaaction currently, so I'm not using m3 and wouldn't be available to solve potential bugs in the refactored code. I'm available again starting June 13th.

**Comment by sircharlo** [comment]
Hey there! I've been using the refactored code and havent seen any issues myself.
I'd probably wait until next week anyways, I don't think anything needs urgent bugfixing
Looking at https://github.com/sircharlo/meeting-media-manager/discussions/315 I think the project itself is in a pretty good place.

Here's my running tally of notable changes between 22.5.7 and the master branch (in no particular order):

- Show keyboard shortcuts first in scene selection menu
- Copy file from local source to media folders right away when uploading to congregation server
- Hide "hide media logo" settings toggle on systems other than Windows
- Clear media folders when changing output folder date format
- Only check for updates if actually online
- Starter a major refactoring of the codebase (thanks 😄)
- Remove non-breaking spaces from filenames to prevent media reading difficulties in rare cases
- Fix for inconsistent alphabetical file sorting on certain systems
- Fix on Linux for when the media presentation window would cause a crash
